src/medLog/medLogger.cpp

```diff
@@ -112,6 +110,9 @@ void medLogger::initializeTeeStreams()
 
 void medLogger::finalizeTeeStreams()
 {
+    QObject::disconnect(this, SIGNAL(newQtMessage(QtMsgType,QString)), this, SLOT(redirectQtMessage(QtMsgType,QString)));
+    qInstallMsgHandler(NULL);
```

Comment:
This code is not related to what the function does (it finalizes the tee streams) so it shouldn't have been moved here.